getting-started-2.rst

```diff
@@ -0,0 +1,473 @@
+
+Return to the :program:`cmd` window on your computer. :kbd:`Control-C` to stop the :program:`grunt` command that popped-open the :program:`Chrome` browser. On Windows, you will be asked to confirm this ``Terminate batch job (Y/N)?``. Enter ``Y`` to confirm.
+
+Connect your device to your computer via USB. Wait for the storage connection to be established (on Windows, this will generally pop up a file browser or an options box that enables you to select a file browser).
+
+At the command prompt, type:
+
+.. code-block:: console
+
+  $ grunt adbpush
+
+.. warning::
+
+  This command will force-close ODK Services, Survey, and Tables, and it will clear all ODK 2.0 data from the device. The data you are pushing will overwrite any exiting application or collected data you might have. Be sure to make backups and be sure you are ready before running this command.
+
+This pushes the configured ODK 2.0 application within this ODK 2.0 Application Designer directory to your device. Because this is a stripped-down version of the Application Designer that only contains the simple demo files, this will copy only those files to the device. When you issue this command, the :program:`cmd` window will display a long series of commands and conclude with a display of overall progress and timings:
+
+.. image:: /img/getting-started-2/geotagger-cmd-gruntpush.*
+  :alt: Geotagger Grunt Push
+
+Now, on your device, launch ODK Tables.
+
+This will initiate the configuration of ODK Tables and conclude with a :guilabel:`Configuration Summary` pop-up reporting that everything was imported successfully. Click :guilabel:`OK`.
+
+Everything should now appear as it did with the application you first joined on https://opendatakit-simpledemo.appspot.com, except you will only have the data rows configured by the ODK 2.0 Application Designer zip, and not any added or modified since that time.
+
+.. _using-odk-2-config-reset-server:
+
+Resetting the Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have the application running on the device, you will typically need to reset the contents of the application server. While the :guilabel:`Reset App Server` button on the device can shuffle the various supporting files between the device and the server, it will not destroy data tables that already exist on the server; this is intentional -- we want to minimize the potential for accidental loss of data.
+
+.. note::
+
+  Whenever you are developing an application, and have found a need to add a new column to an existing table, you will need to manually delete the data tables from the server before using the :guilabel:Reset App Server` button from the device.
```

Comment:
You have left a backtick here in the `:guilabel:` role due to which the test fails. Please fix that!